packages/server/src/integration-test/postgres.spec.ts

```diff
@@ -41,18 +41,17 @@ describe("postgres integrations", () => {
     makeRequest = generateMakeRequest(apiKey, true)
 
     postgresDatasource = await config.api.datasource.create(
-      await databaseTestProviders.postgres.datasource()
+      await getDatasource(DatabaseName.POSTGRES)
     )
   })
 
-  afterAll(async () => {
-    await databaseTestProviders.postgres.stop()
-  })
-
   beforeEach(async () => {
     async function createAuxTable(prefix: string) {
       return await config.createTable({
-        name: `${prefix}_${generator.word({ length: 6 })}`,
+        name: `${prefix}_${generator
```

Comment:
I'm a little concerned about the need to remember that table names can no longer be static, they must be unique across our entire test environment, while this does seem like a major speed improvement, I'm just wondering if theres a better way we could do this.

For most databases we should be able to use a different schema, could this be an option instead of making sure the table names are always unique?

If we must go with unique table names, it would be good to try and centralise this logic somewhere, like a `generateUniqueTableName()` with an explanation of why this function is used, simply to capture this reasoning a bit.

Reply:
I did think a little about this, it _is_ possible to create unique schemas / collections. I can give it a whirl if you'd like. I didn't do it initially because it seemed like a more surprising approach than this one, but an extra layer of safety would be nice.

Reply:
I would probably give it a go - I know its slightly surprising, but at least means if you're copying existing tests to create new ones you don't miss the semi-hidden rule of table names needing to be totally unique across all tests.

Reply:
Not too worried - the table names now don't particularly matter, was just trying to protect against the issue in future!